### PR TITLE
feat: support adding global parameters

### PR DESCRIPTION
**Pull request by CodyTseng**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #2158 

## What is the new behavior?

Support adding global parameters.

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: lib/storages/global-parameter.storage.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  }
+
+  getGlobalParameters() {
+    return this.parameters;
+  }
+
+  cleanGlobalParamters() {
+    this.parameters = [];
+  }
+}
```
Comment (kamilmysliwiec) [inline lib/storages/global-parameter.storage.ts:17]:
```suggestion
export class GlobalParametersStorageHost {
  private parameters = new Array<ParameterObject>();

  add(...parameter: ParameterObject[]) {
    this.parameters.push(...parameter);
  }

  getAll() {
    return this.parameters;
  }

  clear() {
    this.parameters = [];
  }
}
```

Comment (CodyTseng) [reply lib/storages/global-parameter.storage.ts]: Thank you for taking the time to review my commits! I have changed some names as you suggested.

**Comment by kamilmysliwiec** [comment]
LGTM

**Comment by kamilmysliwiec** [comment]
Could you create a PR to the docs as well? 🙏

**Comment by CodyTseng** [comment]
Of course, but my English is not very good, I'll try my best to write about it.

**Pull request merged**